### DISCUSSION TITLE
Create an online rotki demo portal for showing all features

### DISCUSSION THREAD

**Discussion by LefterisJP** [rotki/rotki#6683] [Ideas]
## Abstract

It was suggested to us that it may be hard to figure out what premium can offer and that it may be a nice idea to create a portal where there is a demo account with various addresses and a read only DB in the cloud where users can simply visit, have the account always logged in and allow exploration of the UI with all premium features unlocked.

## Task

Do it. (also probably spec out in more detail)

**Comment by kelsos** [comment]
For the demo, we either need to wrap the docker image or create a custom image with a proxy.

The normal rotki backend should not be directly accessible and the only way to access it would be through the proxy itself.

The purpose of the proxy is to cut access to functionality outside the demo purposes.
e.g.

- Adding/editing accounts
- Viewing and editing API keys and credentials
- It should replace any sensitive information before passing it to the frontend.

With the proxy, we could also potentially mock some stuff that is not easy to display through the usage of public data, such as exchange accounts.
